qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -277,13 +277,13 @@ def _get_value(prop_dict: Dict, prop_name: str) -> Any:
                     or prop_name_map[name] is None
                     or qubits not in prop_name_map[name]
                 ):
-                    logger.debug(
-                        "Gate calibration for instruction %s on qubits %s is found "
-                        "in the PulseDefaults payload. However, this entry is not defined in "
-                        "the gate mapping of Target. This calibration is ignored.",
-                        name,
-                        qubits,
-                    )
+                    # logger.debug(
+                    #     "Gate calibration for instruction %s on qubits %s is found "
+                    #     "in the PulseDefaults payload. However, this entry is not defined in "
+                    #     "the gate mapping of Target. This calibration is ignored.",
+                    #     name,
+                    #     qubits,
+                    # )
```

Comment:
Did you mean to comment this out? I don't actually oppose that since we get a lot of this log, but we should open an issue to investigate why it's happening.

Reply:
Oops nope, I did this to make it easier to debug. I'll change it back and create a separate issue to look into this.